httpd/handler_test.go

```diff
@@ -1036,6 +1029,95 @@ func TestHandler_serveShowMeasurements(t *testing.T) {
 	}
 }
 
+func TestHandler_serveShowTagKeys(t *testing.T) {
+	srvr := OpenServer(NewMessagingClient())
+	srvr.CreateDatabase("foo")
+	srvr.CreateRetentionPolicy("foo", influxdb.NewRetentionPolicy("bar"))
+	srvr.SetDefaultRetentionPolicy("foo", "bar")
+	s := NewHTTPServer(srvr)
+	defer s.Close()
+
+	status, body := MustHTTP("POST", s.URL+`/write`, nil, nil, `{"database" : "foo", "retentionPolicy" : "bar", "points": [
+		{"name": "cpu", "tags": {"host": "server01"},"timestamp": "2009-11-10T23:00:00Z","values": {"value": 100}},
+		{"name": "cpu", "tags": {"host": "server01", "region": "uswest"},"timestamp": "2009-11-10T23:00:00Z","values": {"value": 100}},
+		{"name": "cpu", "tags": {"host": "server01", "region": "useast"},"timestamp": "2009-11-10T23:00:00Z","values": {"value": 100}},
+		{"name": "cpu", "tags": {"host": "server02", "region": "useast"},"timestamp": "2009-11-10T23:00:00Z","values": {"value": 100}},
+		{"name": "gpu", "tags": {"host": "server02", "region": "useast"},"timestamp": "2009-11-10T23:00:00Z","values": {"value": 100}}
+		]}`)
+
+	if status != http.StatusOK {
+		t.Log(body)
+		t.Fatalf("unexpected status after write: %d", status)
+	}
+
+	var tests = []struct {
+		q   string
+		r   *influxdb.Results
+		err string
+	}{
+		// SHOW TAG KEYS
+		{
+			q: `SHOW TAG KEYS`,
+			r: &influxdb.Results{
+				Results: []*influxdb.Result{
+					&influxdb.Result{
+						Rows: []*influxql.Row{
+							&influxql.Row{
+								Name:    "cpu",
+								Columns: []string{"host", "region"},
+							},
+							&influxql.Row{
+								Name:    "gpu",
+								Columns: []string{"host", "region"},
```

Comment:
this should be 
```go
Columns: []string{"tagKey"},
Values: [][]interface{}{{"host"}, {"region"}}
```

I'm also thinking that this could be returned in a single row. With columns `measurement`, `tagKey`. Not sure on this last bit though. Thoughts?